js/private/js_run_binary.bzl

```diff
@@ -107,6 +110,21 @@ def js_run_binary(
 
             This makes node binaries match the expected bazel paradigm.
 
+        run_from_execroot: Run the `tool` js_binary entry_point script that is in the execroot of the output tree
```

Comment:
```suggestion
        run_from_execroot: Run the `tool` js_binary entry_point script in the execroot of the output tree
```

?

Reply:
Actually maybe it is correct now that I read it again, but it feels like a very long sentence that I had to read multiple times

Reply:
Yea. It is tricky to be both precise and to the point with this one.

Reply:
Re-worded to this
```
        run_from_execroot: Run the `entry_point` script of the `js_binary` `tool` that is in the execroot output tree
            instead of the copy that is in runfiles.
```
which I'm not sure if it is any better